opba-auth-rest-impl/src/main/java/de/adorsys/opba/tppauthapi/controller/PsuAuthController.java

```diff
@@ -102,6 +108,23 @@
         return Optional.empty();
     }
 
+    @Override
+    public ResponseEntity<Void> renewalAuthorizationSessionKey(UUID xRequestId, UUID authorizationId) {
+        if (!authorizationKeyFromHttpRequest.isPresent()) {
+            log.warn("No Cookie provied, call not accepted");
+            return new ResponseEntity<>(HttpStatus.UNAUTHORIZED);
+        }
+        String[] cookies = buildAuthorizationCookiesOnAllPaths(authorizationId, authorizationKeyFromHttpRequest.get());
+        String ttl = Long.toString(cookieProperties.getMaxAge().getSeconds());
+        log.info("cookie is renewed for time {} with new value: {}", ttl, cookies[0]);
```

Comment:
Avoid excessively logging, moreover cookie contains secret key that is extremely sensitive and should not be present in logs

Reply:
I changed to log.debug. If secret key is given to client, it may be allowed to log it too?

Reply:
It is clients' secret, administrator shouldn't have access to it. At least I would start with zero-access model

Reply:
ok, just logging, that cookie is renewed. No other info.